scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1826,6 +1908,8 @@ void BPF_STRUCT_OPS(layered_stopping, struct task_struct *p, bool runnable)
 	if (cctx->yielding && used < layer_slice_ns)
 		used = layer_slice_ns;
 	p->scx.dsq_vtime += used * 100 / p->scx.weight;
+	layer->vtime_decay += used * 100 / p->scx.weight;
```

Comment:
I'm not sure this makes sense. Imagine two layers with weight 100 and 200. They have simliar workload compositions but all tasks in the latter layer is niced to have 1/2 per-task weights. This would cancel out the layer weight difference which likely isn't the intention of the configuration.

vtimes and weights usually don't translate well outside of its own timeline. Here, it should be accumulating unadjusted `used` as per-task weights are what direct only how tasks compete within their own layers. As such, `vtime_decay` is probably not the best name. Maybe sth like `usage_avg`?

Also, this is a rather crude way of implementing layer weights. Being a starvation avoidance mechanism, this direction may work out okay but it's likely that there are finer-grained ways to implement this without incurring extra overhead.